rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -206,8 +206,8 @@ def rain(*inputs, rain=None, rain_width = None, rain_height = None, rain_transpa
     rain_image = b.Rain(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
     return (rain_image)
 
-def resize(*inputs, bytes_per_sample_hint=0, image_type=0, interp_type=1, mag_filter= 1, max_size = [0.0, 0.0], min_filter = 1,
-            minibatch_size=32, preserve=False, resize_longer=0.0, resize_shorter= 0.0, resize_x = 0.0, resize_y = 0.0,
+def resize(*inputs, bytes_per_sample_hint=0, image_type=0, interp_type=1, mag_filter= 1, max_size = [], min_filter = 1,
+            minibatch_size=32, preserve=False, resize_longer=0, resize_shorter= 0, resize_x = 0, resize_y = 0, scaling_mode=types.SCALING_MODE_DEFAULT, interpolation_type=types.LINEAR_INTERPOLATION,
```

Comment:
Remove additional spaces before and after = to follow the same style as other parameters
resize_shorter= 0, resize_x = 0, resize_y = 0
mag_filter= 1, max_size = [], min_filter = 1